src/server/pfs/iface.go

```diff
@@ -13,7 +15,7 @@ type APIServer interface {
 
 	NewPropagater(*txncontext.TransactionContext) txncontext.PfsPropagater
 
-	CreateRepoInTransaction(*txncontext.TransactionContext, *pfs_client.CreateRepoRequest) error
+	CreateRepoInTransaction(context.Context, *txncontext.TransactionContext, *pfs_client.CreateRepoRequest) error
```

Comment:
Doesn't the `TransactionContext` have a `context.Context` in it already?

Reply:
We recently removed it.